### ISSUE TITLE
After save on ios location data is lost, android working good

### ISSUE THREAD

**Issue by manarfalah** [react-native-cameraroll/react-native-cameraroll#459]

**Comment by Arjan-Zuidema** [comment]
Hi! I need more info to help you. What code are you using?

**Comment by manarfalah** [comment]
for testing I am selecting an image from gallery this way using react-native-image-crop-picker: (the image have location data)  
```
public selectImageFromGallery = () => {
    ImagePicker.openPicker({
      multiple: false,
      mediaType: 'photo',
      includeExif: true,
      includeBase64: true
    })
      .then((image: any) => {
        this.setState({ image });
        this.saveImageHandler(image);
      })
      .catch(error => { });
  };
```
  
  and here I am saving it back to gallery and the image lost the location data: 
  
 ```
   private SaveImageHandler = (image) => {
     this.savePicture(image.uri || image.path);
  };
  async savePicture(uri) {
    if (Platform.OS === 'android' && !(await this.hasAndroidPermission())) {
      return;
    }
    CameraRoll.save(uri, { album: "Agroscout" });
  }
```

not that on android it is keeping location data, only on ios it is not

**Comment by Yooooomi** [comment]
I'm having the same issue. It seems that it has been resolved in the past, then has be broken again. Somebody with a bit of objective c knowledge can apply the fix again from #134?
Many thanks
